src/data.js

```diff
@@ -8283,6 +8283,25 @@ module.exports = [
       'PHP',
       'SQL',
     ],
+  },{
+    name: 'Kevin Gimbel',
+    description: 'DevOps Engineer who loves Rust + web dev',
+    url: 'https://kevingimbel.de/uses',
+    twitter: '',
```

Comment:
"twitter" is not allowed to be empty

Reply:
Oh, so I can't join if I don't have a twitter account anymore?

Reply:
Sorry for the delay, `twitter: '',` is invalid, but if you remove the line it is valid.

Reply:
I see, I removed it thanks! :)